_posts/2024-06-12-branching-and-versioning.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Explaining the branching and versioning strategy'
+date: 2024-06-23
+tags: branching versioning
+synopsis: 'How does quarkus use Git and the implication on releases'
+author: geoand
+---
+:imagesdir: /assets/images/posts/branching-and-versioning
+
+We often get question about how the Quarkus maintainers handle multiple branches and how this impacts releases. This short post attempts to provide the necessary information for answering such questions:
+
+== Development branch
+
+When a maintainer or contributor wants to make a change to Quarkus source code, a Pull Request is opened against the `main` branch.
+This means that all changes once merged are part of the `main` branch and there are no feature branches (a practice that is sometimes referred to as trunk based development).
```

Comment:
Should we mention the magic 999-SNAPSHOT version?

Reply:
Sure yeah, I'll add that